x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_about_rule/index.test.tsx

```diff
@@ -0,0 +1,165 @@
+    ).toBeTruthy();
+    expect(
+      wrapper
+        .find('EuiFormRow[data-test-subj="detectionEngineStepAboutRuleDescription"] EuiTextArea')
+        .at(0)
+        .prop('isInvalid')
+    ).toBeTruthy();
+  });
+
+  test('it prevents user from clicking continue if no `name` defined', () => {
+    const wrapper = mount(
+      <ThemeProvider theme={theme}>
+        <StepAboutRule
+          addPadding={true}
+          defaultValues={stepAboutDefaultValue}
+          descriptionColumns="multi"
+          isReadOnlyView={false}
+          isLoading={false}
+          setForm={jest.fn()}
+          setStepData={jest.fn()}
+        />
+      </ThemeProvider>
+    );
+
+    const descriptionInput = wrapper
+      .find('textarea[aria-describedby="detectionEngineStepAboutRuleDescription"]')
+      .at(0);
+    descriptionInput.simulate('change', { target: { value: 'Test description text' } });
+
+    const nameInput = wrapper
+      .find('input[aria-describedby="detectionEngineStepAboutRuleName"]')
+      .at(0);
+    const nextButton = wrapper.find('button[data-test-subj="about-continue"]').at(0);
+    nextButton.simulate('click');
+
+    expect(
+      wrapper
+        .find('textarea[aria-describedby="detectionEngineStepAboutRuleDescription"]')
+        .at(0)
+        .props().value
+    ).toEqual('Test description text');
+    expect(nameInput.props().value).toEqual('');
+    expect(
+      wrapper
+        .find('EuiFormRow[data-test-subj="detectionEngineStepAboutRuleName"] label')
+        .at(0)
+        .hasClass('euiFormLabel-isInvalid')
+    ).toBeTruthy();
+    expect(
+      wrapper
+        .find('EuiFormRow[data-test-subj="detectionEngineStepAboutRuleName"] EuiFieldText')
+        .at(0)
+        .prop('isInvalid')
+    ).toBeTruthy();
+  });
+
+  test('it allows user to click continue if `name` and `description` are defined', () => {
+    const wrapper = mount(
+      <ThemeProvider theme={theme}>
+        <StepAboutRule
+          addPadding={true}
+          defaultValues={stepAboutDefaultValue}
+          descriptionColumns="multi"
+          isReadOnlyView={false}
+          isLoading={false}
+          setForm={jest.fn()}
+          setStepData={jest.fn()}
+        />
+      </ThemeProvider>
+    );
+
+    const descriptionInput = wrapper
+      .find('textarea[aria-describedby="detectionEngineStepAboutRuleDescription"]')
+      .at(0);
+    descriptionInput.simulate('change', { target: { value: 'Test description text' } });
+
+    const nameInput = wrapper
+      .find('input[aria-describedby="detectionEngineStepAboutRuleName"]')
+      .at(0);
+    nameInput.simulate('change', { target: { value: 'Test name text' } });
+
+    const nextButton = wrapper.find('button[data-test-subj="about-continue"]').at(0);
+    nextButton.simulate('click');
+  });
+
+  describe('advanced settings', () => {
+    test('it renders advanced settings collapsed initially', () => {});
+
+    test('it expands to show advanced fields', () => {});
```

Comment:
What is up with these tests that don't do anything? Did you want them to do something?

I bet the CI is going to complain about them, but I would remove them. Empty/dead tests aren't any good.

Reply:
Thanks for catching that. I'd put them there early on as placeholders and forgot to remove them. Removed in latest commit.